open_samus_returns_rando/pickup.py

```diff
@@ -146,7 +169,11 @@ def patch_model(self, model_names: list[str], bmsad: dict) -> None:
                 fx_create_and_link["Param1"]["value"] = "leak"
                 fx_create_and_link["Param2"]["value"] = "actors/items/adn/fx/adnleak.bcptl"
                 fx_create_and_link["Param13"]["value"] = True
+            else:
+                bmsad["components"].pop("FX")
+                bmsad["sound_fx"] = ListContainer([])
         else:
+            bmsad["components"].pop("FX")
```

Comment:
I wonder if we could add this here too `bmsad["sound_fx"] = ListContainer([])`

Reply:
In vanilla that's the case for majors iirc. Minors (at least missile tanks) have the stuff in there which is in our template_bmsad.json